internal/overlord/servstate/handlers.go

```diff
@@ -183,7 +187,8 @@ func (m *ServiceManager) doStop(task *state.Task, tomb *tomb.Tomb) error {
 
 	active, ok := m.services[req.Name]
 	if !ok {
-		return fmt.Errorf("service %q is not active", req.Name)
+		// Already stopped
```

Comment:
Should we log something in the task itself (goes into the console), so it is at least known that this was a no-op?